WeakAurasOptions/OptionsFrames/TextEditor.lua

```diff
@@ -229,33 +230,62 @@ local function ConstructTextEditor(frame)
   local dropdown = CreateFrame("Frame", "SettingsMenuFrame", settings_frame, "UIDropDownMenuTemplate")
 
   local function settings_dropdown_initialize(frame, level, menu)
-    for k, v in pairs(editor_themes) do
-      local item = {
-        text = k,
-        isNotRadio = false,
-        checked = function()
-          return WeakAurasSaved.editor_theme == k
-        end,
-        func = function()
-          WeakAurasSaved.editor_theme = k
-          set_scheme()
-          editor.editBox:SetText(editor.editBox:GetText())
-        end
-      }
-      UIDropDownMenu_AddButton(item)
+    if level == 1 then
+      for k, v in pairs(editor_themes) do
+        local item = {
+          text = k,
+          isNotRadio = false,
+          checked = function()
+            return WeakAurasSaved.editor_theme == k
+          end,
+          func = function()
+            WeakAurasSaved.editor_theme = k
+            set_scheme()
+            editor.editBox:SetText(editor.editBox:GetText())
+          end
+        }
+        UIDropDownMenu_AddButton(item, level)
+      end
+      UIDropDownMenu_AddButton(
+        {
+          text = L["Bracket Matching"],
+          isNotRadio = true,
+          checked = function()
+            return WeakAurasSaved.editor_bracket_matching
+          end,
+          func = function()
+            WeakAurasSaved.editor_bracket_matching = not WeakAurasSaved.editor_bracket_matching
+          end
+        },
+      level)
+      UIDropDownMenu_AddButton(
+        {
+          text = L["Tab Spaces"],
+          hasArrow = true,
+          notCheckable = true,
+          menuList = "spaces"
+        },
+      level)
+    elseif menu == "spaces" then
+      local spaces = {2,4,6}
```

Comment:
6?

Anyone in favor of 6 spaces indentation needs to be prevented from using WeakAuras.
I think the 3rd most common indentation size is 3, but having just 2 and 4 is imho good enough.